nf1_cellpainting_env.yml

```diff
@@ -28,5 +28,6 @@ dependencies:
 - conda-forge::wxpython=4.1.0
 - conda-forge::sentry-sdk=0.18.0
 - pip:
+    - cellprofiler==4.2.4
```

Comment:
Could this project use `cellprofiler==4.2.6` (referencing [CellProfiler releases](https://github.com/CellProfiler/CellProfiler/releases))? I recognize `v4.2.4` was possibly referenced at an earlier point in the git history, but it might be good software gardening to see how updating this dependency could benefit things (speed, accuracy, or other capabilities come to mind). Alternatively, if it'd be unwise to update or it incurs unusual challenges, consider placing a comment above this line stating why `v4.2.4` is preferred.

Reply:
Hmm good point. I have no idea if there will be any issues incurred, but I think that it could be good to state `>=4.2.4` since we did use 4.2.4 but it shouldn't hurt anything to use the higher versions. I only worry about this if 5.0 comes out which might need to be tested or blocked in this environment.

I think if we used 4.2.4, then we shouldn't let someone downgrade.

Thank you for this suggestion, I will add the `>=` so that it will get the latest version if available.